doc/content/bib/articles_using_tmap8.bib

```diff
@@ -1,3 +1,14 @@
+@article{FRANKLIN2025115128,
+    title = {{New developments and verification of fusion blanket simulation capabilities in the MOOSE framework}},
```

Comment:
```suggestion
    title = {New developments and verification of fusion blanket simulation capabilities in the {MOOSE} framework},
```

Since none of these are capitalized except for MOOSE and the first word (which bibtex always preserves in my experience), the brackets only really need to be around MOOSE.